src/fabric/src/fabric.erl

```diff
@@ -86,12 +87,21 @@ get_db_info(DbName) ->
     fabric_db_info:go(dbname(DbName)).
 
 %% @doc the number of docs in a database
--spec get_doc_count(dbname()) ->
-    {ok, non_neg_integer()} |
+%% @equiv get_doc_count(DbName, <<"_all_docs">>)
```

Comment:
IRC dialyzer don't understand edoc syntax. Not a big deal for me, feel free to ignore.

Reply:
dunno about dialyzer, but we have `@equiv` everywhere in fabric for such cases